.gitignore

```diff
@@ -2,3 +2,6 @@ tests/go.stripe/ff/customer_ffjson.go
 tests/goser/ff/goser_ffjson.go
 tests/types/ff/everything_ffjson.go
 tests/ff_ffjson.go
+
+# Vim temporaries
```

Comment:
Not a big thing, but isn't editor specific workarounds a bit specific? It should be possible for you to set up a global gitignore for that.